drafts/WEP0001.rst

```diff
@@ -0,0 +1,282 @@
+  they have some understanding of what the conference is about?
+
+After all scores are completed, the grant review team will discuss the top
+applications, prioritized by underrepresented/marginalized groups first,
+score second, and grant them unless discussion leads to another conclusion,
+until funds run out. The review team is not required to always follow the
+score based order. This also depends on the remaining resources.
+
+Speakers will bypass the review process and are immediately accepted, except
+in exceptional circumstances.
+
+Notifying accepted applicants
+-----------------------------
+
+* All accepted applicants will be notified by the grant review team.
+* Applicants need to confirm they are still intending to participate
+  within one week. This term can be extended if requested in the
+  application.
+* If they withdraw their application, the grant review team may be able
+  provide a few extra applicants with grants, prioritized by score.
+* Applicants that are very low on the list, and basically certain not to
+  receive a grant, will be notified by the grant review team as soon as
+  the selection is complete.
+* When all accepted applicants are confirmed, the remaining not accepted
+  applicants will be notified too.
+
+Handling grants and funds
+-------------------------
+
+* Accepted applicants are encouraged to complete their bookings as soon
+  as possible.
+* Some may need to request a visa. In that case, the grant review team
+  should stress that they should not book non-refundable travel or
+  accommodation until their visa is confirmed. Upon request, an invitation
+  letter will be provided, which should mention that costs are covered
+  by the conference, as this can help with the visa application.
+* Applicants need to provide booking confirmations for their costs.
+  Upon receipt of the confirmations (which need to match their legal name),
+  their costs will be refunded as soon as possible (within a few weeks).
+  There may be costs involved in these transfers, which should be covered
+  by the conference.
+* In circumstances where it is not realistically possible, or prohibitively
+  expensive, to transfer the funds, the refund can happen at the conference.
+  This should be an exceptional case though, as not everyone can afford to
+  miss these funds so long.
+* In very exceptional circumstances, transfers may be made before booking
+  confirmations are received. This applies to people that simply can not
+  get the funds together to pay themselves in advance. This requires at least:
+  - Complete details of what expense needs to be made and what for
+  - Being a known member in the community, or a known and well trusted
+    community member vouching for the applicant.
+  Transfers of funds before booking confirmations are received are always
+  at the discretion of the conference organisers.
+  An alternative option is for conference organisers to pay for or book
```

Comment:
I'd like this to be the exception, not the rule.

Reply:
It should be - I will clarify the wording.

Reply:
The wording has been clarified.